charts/newrelic-pixie/templates/_helpers.tpl

```diff
@@ -0,0 +1,142 @@
+{{/* vim: set filetype=mustache: */}}
+{{/*
+Expand the name of the chart.
+*/}}
+{{- define "newrelic-pixie.name" -}}
+{{- default .Chart.Name .Values.nameOverride | trunc 63 | trimSuffix "-" -}}
+{{- end -}}
+
+{{- define "newrelic-pixie.namespace" -}}
+{{- if .Values.namespace -}}
+    {{- .Values.namespace -}}
+{{- else -}}
+    {{- .Release.Namespace | default "pl" -}}
+{{- end -}}
+{{- end -}}
+
+{{/*
+Create a default fully qualified app name.
+We truncate at 63 chars because some Kubernetes name fields are limited to this (by the DNS naming spec).
+*/}}
+{{- define "newrelic-pixie.fullname" -}}
+{{- $name := default .Chart.Name .Values.nameOverride -}}
+{{- if ne $name .Release.Name -}}
+{{- printf "%s-%s" .Release.Name $name | trunc 63 | trimSuffix "-" -}}
+{{- else -}}
+{{- printf "%s" $name | trunc 63 | trimSuffix "-" -}}
+{{- end -}}
+{{- end -}}
+
+{{/* Generate basic labels */}}
+{{- define "newrelic-pixie.labels" }}
+app: {{ template "newrelic-pixie.name" . }}
+app.kubernetes.io/name: {{ include "newrelic-pixie.name" . }}
+chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
+heritage: {{.Release.Service }}
+release: {{.Release.Name }}
+{{- end }}
+
+{{- define "newrelic-pixie.cluster" -}}
+{{- if .Values.global -}}
+  {{- if .Values.global.cluster -}}
+      {{- .Values.global.cluster -}}
+  {{- else -}}
+      {{- .Values.cluster | default "" -}}
+  {{- end -}}
+{{- else -}}
+  {{- .Values.cluster | default "" -}}
+{{- end -}}
+{{- end -}}
+
+{{- define "newrelic-pixie.nrStaging" -}}
+{{- if .Values.global }}
+  {{- if .Values.global.nrStaging }}
+    {{- .Values.global.nrStaging -}}
+  {{- end -}}
+{{- else if .Values.nrStaging }}
+  {{- .Values.nrStaging -}}
+{{- end -}}
+{{- end -}}
+
+{{- define "newrelic-pixie.licenseKey" -}}
+{{- if .Values.global}}
+  {{- if .Values.global.licenseKey }}
+      {{- .Values.global.licenseKey -}}
+  {{- else -}}
+      {{- .Values.licenseKey | default "" -}}
+  {{- end -}}
+{{- else -}}
+    {{- .Values.licenseKey | default "" -}}
+{{- end -}}
+{{- end -}}
+
+{{- define "newrelic-pixie.apiKey" -}}
+{{- if .Values.global}}
+  {{- if .Values.global.apiKey }}
+      {{- .Values.global.apiKey -}}
+  {{- else -}}
+      {{- .Values.apiKey | default "" -}}
+  {{- end -}}
+{{- else -}}
+    {{- .Values.apiKey | default "" -}}
+{{- end -}}
+{{- end -}}
+
+{{/*
+Return the customSecretName where the New Relic license and Pixie API key will be stored
```

Comment:
"Will be" or "are currently being"?

Reply:
@roobre this `newrelic-pixie.customSecretName` is meant to allow a customer that already has a secret created in the cluster and want to reuse it. It's not meant to create a secret with such name.

Reply:
I'll clarify this in the README